tests/functional/trace_fio.py

```diff
@@ -0,0 +1,577 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from math import isclose
+import pytest
+from datetime import timedelta
+from attotime import attotimedelta
+
+from test_utils.size import Size, Unit
+from test_utils.os_utils import Udev
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_result import FioResult
+from storage_devices.disk import DiskType, DiskTypeSet
+from core.test_run import TestRun
+from test_tools.disk_utils import Filesystem
+from test_utils.os_utils import sync
+from test_tools.fs_utils import (
+    create_directory,
+    create_file,
+    move,
+    remove,
+    write_file,
+    ls,
+)
+from api.iotrace_stats_parser import (
+    DeviceTraceStatistics,
+    DeviceStats,
+    IOStatistics,
+    TimeStat,
+    SectorStat,
+    ServiceStats,
+    Percentiles,
+    Metrics,
+)
+import time
+
+mountpoint = "/mnt"
+tolerance_threshold = 0.1
+latency_tolerance_threshold = 10000
+
+
+@pytest.mark.require_disk("traced_dev", DiskTypeSet([DiskType.optane, DiskType.nand]))
+def test_fs_operations():
+    """
+        title: Test for generated statistics correctness
+        description: |
+          Generate workload usign fio and compare fio output with iotracer output.
+        pass_criteria:
+          - Output generated by iotracer matches with workload triggered with fio.
+    """
+    Udev.disable()
```

Comment:
Is this necessary?

Reply:
Fio is writing to raw device and if udev detects writes to disk's first sectors it can trigger additional IO. To avoid this 'polution' I disabled udev.

Reply:
Since fio writes to raw device, udev can trigger some additional io. Udev is disabled to avoid this pollution.